contrib/signet/generate.py

```diff
@@ -0,0 +1,420 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import argparse
+import base64
+import json
+import logging
+import struct
+import sys
+import time
+import subprocess
+
+from binascii import unhexlify
+from io import BytesIO
+
+sys.path.insert(0, "../test/functional/")
```

Comment:
```suggestion
sys.path.insert(0, "../../test/functional/")
```

in my case, it was wrong path.
```
cd ./contrib/signet/
./generate.py --help
```

Reply:
Yes, the paths are a bit confusing. It is meant to be run from main source root. See for example https://en.bitcoin.it/wiki/Signet:Custom:Script

Reply:
Should be fixed